### ISSUE TITLE
Drawer - List structure is applied incorrectly to some resources items?

### ISSUE THREAD

**Issue by joe-replin** [adaptlearning/adapt-contrib-core#109]
## Bug Description:
At the top level of the drawer, related listitems are announced as separate lists of items. This generates non-informative announcements such as "list of 1 item".

## Steps to Reproduce:

    Using a screen reader, activate the resources button.
    Navigate through the top level navigation items in the dialog and note the list structure. 

## Actual Behavior:

Each extension that is placed in the drawer holder is being given a role of "list". And the direct child of each extension is given a role of "listitem".

## Expected Behavior:

I cannot forsee any individual "drawer__item" being given multiple list items. Ensure related list items are structured with role="listitem" under the same role="list". Should the parent "drawer__holder" be given the role of "list" and each "drawer__item" be given the role of "listitem"?
